akka-testkit/src/main/scala/akka/testkit/TestActors.scala

```diff
@@ -19,6 +19,18 @@ object TestActors {
     }
   }
 
+  /**
+   * ForwardActor forwards all messages as-is to specified ActorRef.
+   *
+   * @param ref target ActorRef to forward messages to
+   */
+  class ForwardActor(ref: ActorRef) extends Actor {
+    override def receive = {
+      case message ⇒ ref forward message
+    }
+  }
+
   val echoActorProps = Props[EchoActor]()
+  val forwardActorProps: (ActorRef ⇒ Props) = Props(classOf[ForwardActor], _)
```

Comment:
You don't really save evaluations by using the val - the function it holds must be evaluated each time anyway :-) Memory wise a val needs to stay around where as you just call into a method.

I'm with Endre on this one, make it a `def` please.